Engine/src/Eri/Utils/Events/KeyPress/EventKeyPress.h

```diff
@@ -0,0 +1,127 @@
+#pragma once
+#include "Eri/Core.h"
+
+namespace ERI
+{
+  enum class KeyPressType
+  {
+    UP,
+    DOWN,
+    MAX
+  };
+
+  enum class Keys
```

Comment:
I know this might be future me, but we decided to handle modified keys internally. i.e. shift i is I. But! that still means we need a 'I' key to propagate the correct key to the rest of the system.